src/Symfony/Component/HttpFoundation/Tests/ParameterBagTest.php

```diff
@@ -242,7 +242,11 @@ public function testGetEnumThrowsExceptionWithNotBackingValue()
         $bag = new ParameterBag(['invalid-value' => 2]);
 
         $this->expectException(\UnexpectedValueException::class);
-        $this->expectExceptionMessage('Parameter "invalid-value" cannot be converted to enum: 2 is not a valid backing value for enum "Symfony\Component\HttpFoundation\Tests\Foo".');
+        if (\PHP_VERSION_ID >= 80200) {
```

Comment:
This file change must target a lower branch, 6.0 imho

Reply:
As @MatTheCat mentioned, the bug was added by #48820 in 6.3

And yes, he has found another issue with enums.